ansible/bootstrap.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Let's use the term "provision". I know it may be confusing because of boostrap.sh, but that was used to bootstrap only at the beginning. Now we shall change the naming a bit (other PR for the .sh and user_data).

Also let's add the "async" in the name so `async_provision.yml` sounds good to me?

*also the playbook name"

Reply:
I'm not sure about 'async' in the name, because actually we don't have that with just one task